src/Config/RabbitMQOptions.cs

```diff
@@ -37,6 +37,11 @@ public RabbitMQOptions()
         /// </summary>
         public string Password { get; set; }
 
+        /// <summary>
+        /// Get or sets the VirtualHost used on the RabbitMQ Server.
+        /// </summary>
+        public string VirtualHost { get; set; }
+

```

Comment:
Thanks ! Can you also update the java binding here ?
https://github.com/Azure/azure-functions-rabbitmq-extension/tree/dev/binding-library/java

Please update the release notes in these locations as well accordingly.
https://github.com/Azure/azure-functions-rabbitmq-extension/blob/dev/binding-library/java/release_notes.md
https://github.com/Azure/azure-functions-rabbitmq-extension/blob/dev/release_notes.md

Reply:
I updated my commit.

I'm note sure about the java part, I did what I though was logic.

Reply:
Could you update the tests here as well ?
https://github.com/Azure/azure-functions-rabbitmq-extension/tree/dev/binding-library/java/test/java/com/microsoft/azure/functions/rabbitmq/annotation